src/devices/src/virtio/block/device.rs

```diff
@@ -18,7 +18,7 @@ use std::sync::Arc;
 use logger::{error, warn, IncMetric, METRICS};
 use rate_limiter::{BucketUpdate, RateLimiter};
 use utils::eventfd::EventFd;
-use virtio_gen::virtio_blk::*;
+use virtio_gen::virtio_blk::{VIRTIO_BLK_ID_BYTES, VIRTIO_F_VERSION_1, VIRTIO_BLK_F_FLUSH, VIRTIO_BLK_F_RO, };
```

Comment:
You should squash this commit into the previous one. Ideally we should have two commits, one removing the wildcard usage and another one updating the bindings